tinygrad/runtime/ops_cuda.py

```diff
@@ -47,17 +47,17 @@ class device:
   RawCUDABuffer = RawMallocBuffer
 else:
   import pycuda.autoprimaryctx # pylint: disable=unused-import # noqa: F401
-  import pycuda.driver as cuda # type: ignore
+  import pycuda.driver as pycuda_driver
```

Comment:
just renamed this, helps resolve the naming issue causing the type ignore error on line 53-60

Reply:
yea `cuda` was a bad name